README.md

```diff
@@ -42,8 +42,242 @@ RedHat modifications against the upstream Che include:
 - Keycloak integration
 - [fabric8-analytics Language Server](https://github.com/fabric8-analytics/fabric8-analytics-lsp-server) 
 
+## Developer-friendly scripts to build and deploy or rollout to minishift
+
+The `dev-scripts` folder contains various scripts for developers that allow easily running the Maven
```

Comment:
I find it still complicated. What you should do is.

### How to build it

```bash
## build pre-requisites

## set some env vars

## build
./dev-scripts/build_fabric8.sh
```

`build_fabric8.sh` options:
- `-Dfast`: description...
- 
- 
....